src/orderbook.js

```diff
@@ -125,8 +128,12 @@ function OrderBook(remote,
     });
   }
 
-  function updateFundedAmountsWrapper(transaction) {
-    self.updateFundedAmounts(transaction);
+  function onTransactionWrapper(transaction) {
```

Comment:
I mean instead of making the wrapper function you can do `self._remote.on('transaction', self.onTransaction.bind(self));`